user_guide_src/source/general/urls.rst

```diff
@@ -75,7 +75,7 @@ by different servers, but we will show examples for the two most common web serv
 Apache Web Server
 -----------------
 
-Apache must have the *mod_rewrite* extension enabled. If it does, you can use a ``.htaccess`` file with some simple rules.
+Apache must have the *mod_rewrite* extension enabled. If it has, you can use a ``.htaccess`` file with some simple rules.
```

Comment:
Is this a typo? I don't know.

Reply:
The second sentence is grammatically correct: "Apache must have the mod_rewrite extension enabled. If it has, you can use a .htaccess file with some simple rules."

Here's the breakdown:

"Apache must have the mod_rewrite extension enabled." This is the main statement, indicating a requirement.

"If it has," is a conditional clause beginning with "if," indicating that the action in the main clause ("you can use a .htaccess file") is dependent on the condition stated in the if-clause ("Apache has the mod_rewrite extension enabled").

This construction is grammatically sound and clearly conveys the condition for using the .htaccess file.